src/libcore/atomic.rs

```diff
@@ -1089,3 +1090,23 @@ pub fn fence(order: Ordering) {
         }
     }
 }
+
+macro_rules! impl_Debug {
+    ($($t:ident)*) => ($(
+        #[stable(feature = "atomic_debug", since = "1.3.0")]
+        impl fmt::Debug for $t {
+            fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+                f.debug_tuple(stringify!($t)).field(&self.load(Ordering::Relaxed)).finish()
```

Comment:
I think the safe default for ordering is the most restrictive one (SeqCst), rather than the most relaxed one. It might be slower, but I doubt anybody is caring about performance when building debug strings for, most likely, printing.